src/Harald.WebApi/Features/Connections/Domain/Model/SenderId.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using Harald.WebApi.Domain;
+
+namespace Harald.WebApi.Features.Connections.Domain.Model
+{
+    public class SenderId :StringSubstitutable
+    {
+        public SenderId(string value) : base(value)
+        {
+        }
+        
+        public static explicit operator SenderId(String input) 
```

Comment:
Making these operators implicit, instead of explicit, will save a lot of boxing statements all over our code base and allow us to do more defensive casting @ https://coderwall.com/p/18dgtg/using-as-and-is-for-safe-casting-in-c . Do they really have to be explicit? :)

Reply:
Yes we currently have 7 of those explicit operators. It all started with json serializer that didn't do the conversion when there was no explicit operator. I would like to get rid of all of them but the issue of serializers would need to be addressed first.

Reply:
I think we should drop the JsonSerializer all together. Serialization settings are very contextual and you can easily have remote APIs that serve both Json and Xml (e.g. JAXB enabled endpoints) thus I would say that in case of our facade the serialization settings belong to each individual request class. :)

If we want sensible defaults we can use the static operator on JsonConverter.DefaultSettings.

Reply:
Ok I propose we make a refactoring commit where we take the serializer out and remove the unwanted explicit.